src/components/SelectionList/BaseListItem.tsx

```diff
@@ -10,28 +10,26 @@ import useStyleUtils from '@hooks/useStyleUtils';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 import CONST from '@src/CONST';
-import RadioListItem from './RadioListItem';
 import type {BaseListItemProps, ListItem} from './types';
-import UserListItem from './UserListItem';
 
 function BaseListItem<TItem extends ListItem>({
     item,
-    isFocused = false,
+    wrapperStyle,
+    selectMultipleStyle,
     isDisabled = false,
-    showTooltip,
     shouldPreventDefaultFocusOnSelectRow = false,
     canSelectMultiple = false,
     onSelectRow,
     onDismissError = () => {},
     rightHandSideComponent,
     keyForList,
+    children,
 }: BaseListItemProps<TItem>) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
     const {translate} = useLocalize();
     const isUserItem = 'icons' in item && item?.icons?.length && item.icons.length > 0;
```

Comment:
@burczu Maybe is better to remove all user related variables and move it outside to UserListItem?

Reply:
yeah, this is something I've missed I think - you are right: `BaseListItem` could just expose the `pendingAction` and `errors` props that could be handled by the specific list item components if necessary

Reply:
e.g. each list could now have separated props and will inherit base props from the ListItem type, we could move `errors`, `pendingFields`, `invitedSecondaryLogin` (as FooterComponent or renderFooterComponent) to new UserListItem type

Reply:
yeah, I'm re-shaping types slightly

Reply:
done